components/Post/index.tsx

```diff
@@ -4,9 +4,10 @@ export interface IPost {
   title: string;
   date: string;
   category: string;
-  body: string;
+  body: any;
```

Comment:
This is an object, coming from Contentful. See the RichText type. 

Usually, you transform it to a more React friendly object in the backend code.